core/src/commonMain/kotlin/com/ustadmobile/core/contentformats/xapi/endpoints/XapiStatementEndpointExt.kt

```diff
@@ -29,7 +30,7 @@ fun XapiStatementEndpoint.storeProgressStatement(account: UmAccount, entry: Cont
         }
         this.`object` = XObject().apply {
             this.id = entry.entryId ?: UMFileUtil.joinPaths(account.endpointUrl,
-                    "/contentEntryUid/${entry.contentEntryUid}")
+                    "/contentEntryUid/${entry.contentEntryUid}/$clazzUid/")
```

Comment:
@SamihMustafa I think the xObject id should not have clazzUid. It's the same object, regardless of which class it is being used with. This doesn't stop merging it, but can you please remove this for the next pull request?